### PR TITLE
PPP Draft

### PR DESCRIPTION
**Pull request by danielpappa**
Created PPP, updated some diagrams and cleaned UG, DG and main page

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -393,17 +393,26 @@ Example of usage:
...
+### Delete all transactions at once: `clear` [coming soon]
+Clears all previously entered incomes and expenses. This will reset the .txt file and allow you to start from scratch.
+
 ### Set Password: `setPassword` [coming soon]
-Set new password.
+Set new password to add a layer of security for the tracking of your financial transactions.
 
 ### Change Password: `changePassword` [coming soon]
-Change Password.
+Change the password in case you believe your previous one may have been leaked.
```
Comment (BenedictChannn) [inline docs/UserGuide.md:403]: Clear elaboration

File: docs/team/danielpappalardo.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+* **Features**:
+    * Implemented the basic structure for incomes and the related commands (add, delete and view).
+    * Implemented the basic structure for the income statement and the overall financial statement which comprises both income and expense statement entries.
+    * Added the ability to set, update and delete a budget:
+      * What it does: allows the user to handle his overall cash-flow by specifying a budget. The budget in CashLeh is kept separate from net cash on hand, meaning that is is to be interpreted as some sort of cushion which the user wants to always have at his availability.
+      * Highlights: this implementation comes with an option for the user to view his current budget-level. By calling the appropriate command, both budget and net cash on hand, as well as a progress bar will be displayed.
+* **Project Management:**
+  * Managed releases v1.0 and v2.0 on GitHub
+  * Helped maintaining the issue tracker on GitHub
```
Comment (BenedictChannn) [inline docs/team/danielpappalardo.md]: I believe "maintain" is more appropriate

**Review by BenedictChannn — APPROVED** [review]
LGTM

File: docs/AboutUs.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
...
-| ![](https://via.placeholder.com/100.png?text=Photo) |    Shan YuXuan    | [Github](https://github.com/xuan127)    |  [Portfolio](team/xuan127.md)   |
-| ![](https://via.placeholder.com/100.png?text=Photo) | Daniel Pappalardo |      [Github](https://github.com/)      |  [Portfolio](docs/team/johndoe.md)   |
-| ![](https://via.placeholder.com/100.png?text=Photo) |   Benedict Chan   |      [Github](https://github.com/)      |  [Portfolio](docs/team/johndoe.md)   |
-| ![](https://via.placeholder.com/100.png?text=Photo) |     Kai Ning      |      [Github](https://github.com/)      |  [Portfolio](docs/team/johndoe.md)   |
+| Display                                              |       Name        |                Github Profile                 |              Portfolio               |
+|------------------------------------------------------|:-----------------:|:---------------------------------------------:|:------------------------------------:|
+| ![](https://via.placeholder.com/100.png?text=Photo)  |     Ming Chun     | [Github](https://github.com/mcmc101001)       | [Portfolio](docs/team/mcmc101001.md) |
```
Comment (mcmc101001) [inline docs/AboutUs.md]: i don't think need docs as everyone else does not

**Review by mcmc101001 — CHANGES_REQUESTED** [review]

**Review by mcmc101001 — APPROVED** [review]
LGTM

**Pull request merged**